### PR TITLE
Other fixes

### PR DESCRIPTION
**Pull request by imisi-akande**
#### What does this PR do?
This PR: 
- Replace current Tools with those in https://newslab.withgoogle.com/training/tools
- Removes partners page from nav bar and move them to "Awards Section" on both Index and certification pages.

#### Description of tasks performed
-  Modify the blue-section of the tools page    
- Add a sub-nav bar that contains tools, courses and Advanced learning to the tools page     
- Add Horizontal rule below the sub-nav bar   
- Add the first row of cards on Google tools selection   
- Add subsequent rows of cards on Google tools section  
- Add “follow us on social media” section to the front end page 
- Integrate the changes made in  front end to the course builder

- Remove partners page from nav bar on index and certification page in front-end directory
- Move partners page contents to index page in front-end directory
- Move partners page content to certification page in front-end directory
- Integrate the changes made on front-end directory into course builder

### CODE REVIEW

File: frontend/scss/_common.scss [hunk 1]
```diff
@@ -57,7 +57,10 @@ img {
 }
 
 .blue-section {
-  background-color: $hover-blue;
+  background-image: url(../images/news-lab.png);
```
Comment (tonymontaro) [inline frontend/scss/_common.scss]: Please use a different class for this, the properties you added here will affect other pages (e.g the homepage)

File: frontend/scss/_common.scss [hunk 2]
```diff
@@ -98,3 +101,7 @@ img {
   @extend .section;
   padding: 2rem 0;
 }
+
+.section.mission.blue-section.style-scope.home-page {
+  display: none;
```
Comment (tonymontaro) [inline frontend/scss/_common.scss]: What is the point of hiding this?

File: frontend/scss/homepage/_main.scss [hunk 3]
```diff
@@ -287,3 +288,39 @@
...
+.join {
+  margin-top: 1.8rem;
+  padding-top: 20px;
+  padding-bottom: 50px;
+  background-color: #1477d6;
+  color: #fff;
+  text-align: center;
+}
+
+.join__wrap {
```
Comment (tonymontaro) [inline frontend/scss/homepage/_main.scss]: Please use consistent css naming convensiopn (e.g join-wrap)

File: frontend/scss/pages/_tools.scss [hunk 4]
```diff
@@ -181,3 +187,98 @@
...
+
+.navbar-course li a {
+  color: #000;
+  font-weight: 400;
+  padding-left: 40px;
+}
+
+.hr-sect {
+	display: flex;
+	flex-basis: 100%;
```
Comment (tonymontaro) [inline frontend/scss/pages/_tools.scss]: Ensure proper indentation, we are currently using 2 space indent for scss files.

**Review by tonymontaro — CHANGES_REQUESTED** [review]
Excellent work, please address the issues raised.

File: frontend/scss/_common.scss (hunk 1 above)
Comment (imisi-akande) [reply frontend/scss/_common.scss]: Ok.done

File: frontend/scss/_common.scss (hunk 2 above)
Comment (imisi-akande) [reply frontend/scss/_common.scss]: Class name has been removed from style sheet

File: frontend/scss/homepage/_main.scss (hunk 3 above)
Comment (imisi-akande) [reply frontend/scss/homepage/_main.scss]: Ok. done

File: frontend/scss/pages/_tools.scss (hunk 4 above)
Comment (imisi-akande) [reply frontend/scss/pages/_tools.scss]: Done.

File: frontend/index.html [hunk 5]
```diff
@@ -117,46 +117,55 @@ <h1 class="intro-title">Improve your digital storytelling with a combination of
...
-        </div>
-        <div class="item">
-          <img src="images/award2.png" alt="">
-          <p>
-            European Commission <br> Digital Skills Award 2016
-          </p>
+          <div class="item">
+              <img src="images/hha.gif" alt="">
+              <p>
+                UK Digital Experience Award <br> Customer Experience Innovation
```
Comment (tonymontaro) [inline frontend/index.html]: Awesome work here @andela-iakande, we need to change the text below the images to the name of the respective partners.

**Review by tonymontaro — CHANGES_REQUESTED** [review]

File: frontend/index.html (hunk 5 above)
Comment (andela-angene) [reply frontend/index.html]: Let's also add these changes to coursebuilder/modules/explorer/_static/components/pages/homepage.html

**Review by tonymontaro — APPROVED** [review]

**Pull request merged**